controllers/ibmvpccluster_controller.go

```diff
@@ -220,27 +198,19 @@ func (r *IBMVPCClusterReconciler) reconcileDelete(clusterScope *scope.ClusterSco
 		return ctrl.Result{RequeueAfter: 1 * time.Minute}, nil
 	}
 
-	if clusterScope.IBMVPCCluster.Spec.ControlPlaneLoadBalancer != nil {
-		deleted, err := clusterScope.DeleteLoadBalancer()
-		if err != nil {
-			return ctrl.Result{}, errors.Wrap(err, "failed to delete loadBalancer")
-		}
-		// skip deleting other resources if still have loadBalancers running.
-		if deleted {
-			return ctrl.Result{RequeueAfter: 1 * time.Minute}, nil
-		}
+	deleted, err := clusterScope.DeleteLoadBalancer()
```

Comment:
why this `if clusterScope.IBMVPCCluster.Spec.ControlPlaneLoadBalancer != nil {` is removed from here?

Reply:
@mkumatag now that we are removing FIP approach, the load balancer should be the default way (so that would mean we assume it to be non-nil), if its nil throw validation error at the start itself. Please correct me if I have misunderstood something here.

Reply:
@mkumatag is your understanding that `ControlPlaneEndpoint` field will serve as alternative to this one (ControlPlaneLoadBalancer), incase user decides to not create one and plugin directly the values via ControlPlaneEndpoint ?

Reply:
@mkumatag just wanted your thoughts on above comments before we close this one.

Reply:
yes, user can create something upfront and use that instead of creating a new one.